crypto/refcount_c11.c

```diff
@@ -26,12 +26,12 @@
 
 
 // See comment above the typedef of CRYPTO_refcount_t about these tests.
-static_assert(alignof(CRYPTO_refcount_t) == alignof(_Atomic CRYPTO_refcount_t),
+_Static_assert(alignof(CRYPTO_refcount_t) == alignof(_Atomic CRYPTO_refcount_t),
```

Comment:
We use [OPENSSL_STATIC_ASSERT](https://github.com/aws/aws-lc/blob/7d3101fb08892c464bdf1ca3a4d4cbea5347820a/crypto/refcount_win.c#L23) other places for this same purpose. It's defined here: https://github.com/aws/aws-lc/blob/7d3101fb08892c464bdf1ca3a4d4cbea5347820a/include/openssl/type_check.h#L67-L98